includes/wcdn-template-functions.php

```diff
@@ -323,28 +328,37 @@ function wcdn_get_order_info( $order, $type = '' ) {
 			);
 		}
 	}
-
-	if ( 'invoice:' === wcdn_get_template_type() ) {
-		if ( isset( $data['invoice_date']['active'] ) ) {
-			$label                  = $data['invoice_date']['invoice_date_text'];
-			$fields['invoice_date'] = array(
-				'label'       => __( $label, 'woocommerce-delivery-notes' ), // phpcs:ignore
-				'value'       => wcdn_get_order_invoice_date( $wdn_order_id ),
-				'font-size'   => $data['invoice_date']['invoice_date_font_size'],
-				'font-weight' => $data['invoice_date']['invoice_date_style'],
-				'color'       => $data['invoice_date']['invoice_date_text_colour'],
-				'active'      => 'yes',
-			);
+	
+	if ( isset( $data['invoice_date']['active'] ) ) {
+		if ( isset( $data['invoice_date']['invoice_date_text'] ) && ! empty( $data['invoice_date']['invoice_date_text'] ) ) {
+			$label = $data['invoice_date']['invoice_date_text'];
 		} else {
-			$fields['invoice_date'] = array(
-				'label' => __( 'Invoice Date', 'woocommerce-delivery-notes' ),
-				'value' => wcdn_get_order_invoice_date( $wdn_order_id ),
-			);
+			$label = __( 'Invoice Date', 'woocommerce-delivery-notes' );
 		}
+		$label                  = $label;
+		$fields['invoice_date'] = array(
+			'label'       => __( $label, 'woocommerce-delivery-notes' ), // phpcs:ignore
+			'value'       => wcdn_get_order_invoice_date( $wdn_order_id ),
+			'font-size'   => $data['invoice_date']['invoice_date_font_size'],
+			'font-weight' => $data['invoice_date']['invoice_date_style'],
+			'color'       => $data['invoice_date']['invoice_date_text_colour'],
+			'active'      => 'yes',
+		);
+	} else {
+		$fields['invoice_date'] = array(
+			'label' => __( 'Invoice Date', 'woocommerce-delivery-notes' ),
+			'value' => wcdn_get_order_invoice_date( $wdn_order_id ),
+		);
 	}
+	
 
 	if ( isset( $data['order_number']['active'] ) ) {
-		$label                  = $data['order_number']['order_number_text'];
+		if ( isset( $data['order_number']['order_number_text'] ) && ! empty( $data['order_number']['order_number_text'] ) ) {
+			$label = $data['order_number']['order_number_text'];
+		} else {
+			$label = __( 'Order Number', 'woocommerce-delivery-notes' );
+		}
+		$label                  = $label;
```

Comment:
Why has this redundant line of code been added? It's present at multiple places. Please remove it.